.travis.yml

```diff
@@ -40,9 +40,9 @@ jobs:
         - <<: *test-matrix
           env: DMD=2.078.3.s* F=devel
         - <<: *test-matrix
-          env: DMD=2.086.* F=production
+          env: DMD=2.087.* F=production
```

Comment:
Just one question: bumping the DMD version is to confirm that the bug fixes in the previous commits work, right?

Reply:
(If so, maybe just add that to the commit message.)

Reply:
No, it's because Matthias told me to do it :)

Reply:
Do you know why he suggested that?

Reply:
I'm just thinking as this is a patch release we should minimise changes.

Reply:
The conversation with him is higher up in this PR. If it would make it simpler to review/accept, I can remove that commit. Should I do that?

Reply:
Hmm, I don't see a conversation, just one comment (https://github.com/sociomantic-tsunami/dhtproto/pull/214#issuecomment-519106682). Let's leave it for now until Matthias can explain himself.

Reply:
> I'm just thinking as this is a patch release we should minimise changes.
Good point. Should be updated earliest in the next minor.
@don-clugston-sociomantic can you please remove the CI change ;). (Sorry for the unnecessary noise)

Reply:
Now it is back to the original :)